phys/module_sf_pxlsm.F

```diff
@@ -1593,7 +1600,8 @@ SUBROUTINE QFLUX (DENS1,  QV1,   TA1,  RG,     RAW, QSS,           & ! in
 !      IF (QSS .LT. QV1) BETAP = 1.0  
       IF (QSS .LT. QV1) RSOIL = 0.0
 !      EG = DENS1 * (1.0 - VEGFRC) * BETAP * (QSS - QV1) / RAW
-      EG = DENS1 * (1.0 - VEGFRC) * (QSS - QV1) / (RAW + RSOIL)
+!      EG = DENS1 * (1.0 - VEGFRC) * (QSS - QV1) / (RAW + RSOIL)
```

Comment:
Please remove the old code, not just comment it out

Reply:
Dave, this goes for other comments as well. We'll look over both ACM2 and PX LSM and make sure it complies with these standards. There are probably other instances. We'll scrub both codes better. Apologies for this legacy stuff.